src/frontend/packages/core/src/components/users/row/base/RowUserBase.test.tsx

```diff
@@ -17,8 +17,8 @@ describe('RowUserBase', () => {
   it('basic render with empty Box instead of Menu if there is not actions defined', async () => {
```

Comment:
Squash this commit with the previous commit.

Reply:
Done